src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -873,7 +869,15 @@ def generic_update_command(self, name,
             child_arg_name=child_arg_name,
             **merged_kwargs)
 
-    def generic_wait_command(self, name, getter_name='get', getter_type=None, **kwargs):
+    def wait_command(self, name, getter_name='get', getter_type=None, **kwargs):
+        self._generic_wait_command(name, getter_name=getter_name, getter_type=getter_type,
```

Comment:
I'd recommend you get rid of getter type for these since the type is specified by whether `wait_command` or `custom_wait_command` is used.